src/appleseed.shaders/src/appleseed/as_glass.osl

```diff
@@ -87,21 +87,6 @@ shader as_glass
         int gafferNoduleLayoutVisible = 0,
         int divider = 1
     ]],
-    string in_distribution = "ggx"
```

Comment:
To be honest, I hadn't noticed that these parameters were marked as deprecated. That makes them less of a problem than I initially thought.